TopicsImplementations/src/topiczerotwo/SQLiteConnection.java

```diff
@@ -0,0 +1,63 @@
+package topiczerotwo;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class SQLiteConnection implements SQLConnector {
```

Comment:
The three implementations of SQLConnector have a lot of common code. Maybe you can create an abstract class that implements SQLConnector holding that common code and adding a few hook methods to be implemented in each child of the abstract class. Then the three implementations could extend the abstract class instead of implementing the interface